Assets/BossRoom/Scripts/Shared/Net/UnityServices/Lobbies/LobbyServiceFacade.cs

```diff
@@ -126,7 +127,7 @@ void OnSuccess(Lobby lobby)
                     }
                     m_UnityServiceErrorMessagePub.Publish(new UnityServiceErrorMessage("Host left the lobby","Disconnecting."));
                     ForceLeaveLobbyAttempt();
-                    m_ApplicationController.QuitGame();
+                    // no need to disconnect NGO, it should already be handled by NGO callback to disconnect

```

Comment:
Just nitpick of NGO->Netcode or Netcode for GameObjects. Good to go otherwise!

Reply:
done